changelog/_unreleased/2024-04-26-fix-openapi-validation.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Fix OpenApi Schema validation
+issue: NEXT-00000
+author: Benjamin Wittwer
+author_email: benjamin.wittwer@a-k-f.de
+author_github: akf-bw
+---
+# Core
+* Changed `01-lint-openapi` to run correctly & cache composer files
+* Changed `CalculatedPrice` API Schema to correctly use `oneOf` with `null`
+* Changed `Product` API Schema to correctly use `oneOf` with `null`
+* Changed `ReferencePrice` API Schema to correctly use `oneOf` with `null`
```

Comment:
thank you for this 💙 
from now on, we will have `calculatedPrice: calculatedPrice | null` and so on for the entities which are allowed to be null (and also present in the response).

@shyim could you have a look at the CI improvement as well?